Sources/Plasma/Apps/plUruLauncher/plClientLauncher.cpp

```diff
@@ -109,6 +109,7 @@ void plShardStatus::Run()
     curl_easy_setopt(curl.get(), CURLOPT_ERRORBUFFER, fCurlError);
     curl_easy_setopt(curl.get(), CURLOPT_USERAGENT, "UruClient/1.0");
     curl_easy_setopt(curl.get(), CURLOPT_URL, url.c_str());
+    curl_easy_setopt(curl.get(), CURLOPT_FOLLOWLOCATION, 1);
```

Comment:
Should probably also set `CURLOPT_MAXREDIRS` so we don't risk spinning forever in a redirect loop.

Reply:
Ugh, of course that has an unsafe default